### PR TITLE
refactor: complete seed column definitions and move location

### PR DESCRIPTION
**Pull request by lawrenceadams**
### Sumamry

This pull request includes significant updates to the `seeds` configuration in the `dbt_project.yml` file and the addition of detailed seed configurations in the `seeds/synthea/_sources.yml` file. The main changes involve enabling the `synthea` schema and adding comprehensive column type definitions for various tables.

Updates to `dbt_project.yml`:

* Removed table schema definitions.

Additions to `seeds/synthea/_sources.yml`:

* Added detailed seed configurations with column type definitions for synthea output.

### Why Bother?

 - Removes ambiguity
 - If using other seed files/larger seed files columns may not be sniffed in their entirety and this will result in errors downstream when an incorrect type (e.g. ZIP looks like a `numeric` type by default - causing issues when doing string parsing)

### CODE REVIEW

File: seeds/synthea/_sources.yml [hunk 1]
```diff
@@ -0,0 +1,303 @@
...
+        HEALTHCARECLAIMTYPEID2: integer
+  - name: claims_transactions
+    config:
+      column_types:
+        ID: varchar
+        CLAIMID: varchar
+        CHARGEID: integer
+        PATIENTID: varchar
+        TYPE: varchar
+        AMOUNT: numeric
```
Comment (katy-sadowski) [inline seeds/synthea/_sources.yml:74]: `float`, for consistency with other dollar amounts?

File: seeds/synthea/_sources.yml [hunk 2]
```diff
@@ -0,0 +1,303 @@
...
+        UNITS: integer
+        DEPARTMENTID: integer
+        NOTES: varchar
+        UNITAMOUNT: numeric
+        TRANSFEROUTID: integer
+        TRANSFERTYPE: varchar
+        PAYMENTS: numeric
+        ADJUSTMENTS: varchar
+        TRANSFERS: numeric
+        OUTSTANDING: numeric
```
Comment (katy-sadowski) [inline seeds/synthea/_sources.yml:95]: all should be floats - dollar amounts

**Review by katy-sadowski — CHANGES_REQUESTED** [review]
thanks so much for doing this!  suggested a couple minor fixes.

**Comment by lawrenceadams** [comment]
Thank you @katy-sadowski !! Some silly mistake from myself...

**Review by katy-sadowski — APPROVED** [review]
All good!  Thank you!!!

**Pull request merged**